### PR TITLE
Implement crowdloan reward distribution pallet

### PR DESCRIPTION
**Pull request by sudipghimire533**
- install vesting pallet
- write crowdloan reward distribution pallet
- fix benchmarking of `pallet-staking`

### CODE REVIEW

File: pallets/crowdloan-reward/src/functions.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    instant_amount: types::BalanceOf<T>,
+) -> DispatchResult {
+    if instant_amount.is_zero() {
+        Ok(())
+    } else {
+        <T as crate::Config>::Currency::transfer(reward_source, user, instant_amount, ExistenceRequirement::AllowDeath)
+    }
+}
+
+pub fn do_vesting_reward<T: crate::Config>(
```
Comment (penumbra23) [inline pallets/crowdloan-reward/src/functions.rs]: Maybe some comments would be okay on each function, although it's clear but just to point out what it does in one sentence

**Review by penumbra23 — DISMISSED** [review]

File: pallets/crowdloan-reward/src/functions.rs (hunk 1 above)
Comment (sudipghimire533) [reply pallets/crowdloan-reward/src/functions.rs]: Done on commit [`ad3d6ea`](https://github.com/DataHighway-DHX/DataHighway-Parachain/pull/55/commits/ad3d6eaff6630767f8797f7863ebf982b15d0d95)

File: runtime/src/lib.rs
Comment (penumbra23) [inline runtime/src/lib.rs:1]: Increment this one to 9 ?

Comment (sudipghimire533) [reply runtime/src/lib.rs:1]: Done in commit [`0bbea7e`](https://github.com/DataHighway-DHX/DataHighway-Parachain/pull/55/commits/0bbea7ebc30d0b62765a5cc5339e868a43fdf02c)

**Review by penumbra23 — DISMISSED** [review]

**Review by penumbra23 — DISMISSED** [review]

**Review by penumbra23 — APPROVED** [review]

**Review by iTranscend — APPROVED** [review]

**Pull request merged**